### PR TITLE
Backtothefuture 2

### PR DESCRIPTION
**Pull request by blissfulmondays**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -271,5 +271,119 @@ nav {
...
+    font-family: 'Helvetica 25 UltraLight', sans-serif;
+    font-family: 'Helvetica 35 Thin', sans-serif;
+    font-family: 'Helvetica 45 Light', sans-serif;
+    font-family: 'Helvetica 55 Roman', sans-serif;
+    font-family: 'Helvetica 65 Medium', sans-serif;
+}
+
+
+  /***************** Part II Nav Bar & logo ********************/
+  nav {
```
Comment (RenaBMew) [inline styles.css]: This code is already in the nav above.

File: styles.css [hunk 2]
```diff
@@ -271,5 +271,119 @@ nav {
...
+    margin-left: 24px;
+  }
+  
+  .header__logo {
+    margin-right: auto;
+    text-transform: uppercase;
+    max-height: 260px;
+  }
+
+  .intro__p3-logo-p {
```
Comment (RenaBMew) [inline styles.css]: This needs to be specific to your page or it will overwrite the same selector above.

File: styles.css [hunk 3]
```diff
@@ -271,5 +271,119 @@ nav {
...
+    margin-bottom: 96px;
+    box-shadow: 4px 4px 10px var(--gray);
+    background-color: var(--dark-gray);
+    padding-left: 30px;
+  }
+
+ 
+
+
+  .img__p3-cast-box {
```
Comment (RenaBMew) [inline styles.css]: This the same as above, can be removed.

**Review by RenaBMew — COMMENTED** [review]
Please see comments in CSS file.  Some code needs to be adjusted. 

For the HTML page, you may want to update the image selectors with v in them (the vertical images). I took this CSS selector out of styles.css since I thought it looked better without the vertical photos. If you want to keep them on your page you will have to readd those CSS selectors.

File: part2.html [hunk 4]
```diff
@@ -61,135 +61,135 @@ <h1>Synopsis</h1>
...
                             </p>
                         </div>
                     </div>
                 
-                    <div class="img__p2-boxv">
-                        <img class="img__p2-box-imgv" src="photos/filming on blue screen.jpg" alt="Lloyd and Fox behind the scenes.">
-                        <div class="img__p2-captionv">
-                            <p class="img__p2-caption-pv">
-                                Clara inspects a small model of the DeLorean, she has ever seen a car before.
+                    <div class="img__p3-boxv">
```
Comment (RenaBMew) [inline part2.html:173]: img__p3-boxv selector is not in styles.css sheet.

File: styles.css [hunk 5]
```diff
@@ -271,5 +271,119 @@ nav {
...
 
+<<<<<<< HEAD
+/*************************************************************/
+
+/***************** Back to the Future II *********************/
+.body__p2 {
+
+    background-image: url(photos/NBP-grit-13\ copy.jpg);
+    margin: auto;
+    padding: 0 120px 0 120px;
```
Comment (RenaBMew) [inline styles.css]: remove this padding

File: part2.html [hunk 6]
```diff
@@ -8,38 +8,38 @@
...
+    <link rel="stylesheet" href="styles.css">
 
 
     <title>Back to the Future Part II</title>
 </head>
 
 <body class="body__p2">
-        <header class="header">
-            <nav class="header__nav">
-                <a class="header__logo" href="index.html"><img src="photos/bttf2logo.png"> </a>
```
Comment (RenaBMew) [inline part2.html:20]: this logo should be moved down above the run time text.

File: part2.html [hunk 7]
```diff
@@ -61,135 +61,135 @@ <h1>Synopsis</h1>
...
+                <p class="art__p3-p">
                     Avoiding the traditional futuristic metallic look, Johnston used bright "but not fluorescent," colors, and a vast array of fabrics to create outfits for the cast, as well as over 150 extras. Johnston worked closely with production designer Rick Carter to make silre "my costumes balanced his set. The audience has to believe that people routinely go in and out of these buildings, and fit in this environment." As the times have changed, the future has become a truly equal society for men and women, so Johnston felt no qualms about a fashion style that was "absolutely sexist. Men look like men, and women look like women. There is no chance of confusion."
                 </p>
             </article>
         </section>
 
         <section>
-            <article class="art__p2">
+            <article class="art__p3">
                 <h1>Movie Stills</h1>
```
Comment (RenaBMew) [inline part2.html]: h2

File: part2.html [hunk 8]
```diff
@@ -198,62 +198,62 @@ <h1>Movie Stills</h1>
         </section>
 
         <section>
-            <article class="art__p2">
+            <article class="art__p3">
                 <h1>The Arduous Makeup Process</h1>
```
Comment (RenaBMew) [inline part2.html]: h2

File: part2.html [hunk 9]
```diff
@@ -8,38 +8,38 @@
...
-                <a class="header__logo" href="index.html"><img src="photos/bttf2logo.png"> </a>
-                <a class="header__link" href="index.html">Back to the Future</a>
-                <a class="header__link" href="part1.html">Part I</a>
-                <a class="header__link" href="part2.html">Part II</a>
-                <a class="header__link" href="part3.html">Part III</a>
-            </nav>
-        </header>
+         <header>
+        <nav class="navbar">
+            <a class="link navname" href="index.html">Back to the Future</a>
```
Comment (RenaBMew) [inline part2.html:20]: This should say Home

File: styles.css [hunk 10]
```diff
@@ -271,5 +271,119 @@ nav {
     padding: 20px;
 }
 
+<<<<<<< HEAD
+/*************************************************************/
+
+/***************** Back to the Future II *********************/
+.body__p2 {
+
+    background-image: url(photos/NBP-grit-13\ copy.jpg);
```
Comment (RenaBMew) [inline styles.css:279]: https://www.filmstories.co.uk/wp-content/uploads/2019/10/back-to-the-future.jpg

File: part2.html (hunk 9 above)
Comment (blissfulmondays) [reply part2.html:20]: which part should say home, the header?

File: styles.css (hunk 10 above)
Comment (blissfulmondays) [reply styles.css:279]: this is my background texture, not eh back to the future image

Comment (RenaBMew) [reply styles.css:279]: Sorry I typed that fast =/

**Review by RenaBMew — APPROVED** [review]

**Pull request merged**